qa/repository-multi-version/build.gradle

```diff
@@ -29,6 +29,11 @@ BuildParams.bwcVersions.withIndexCompatible { bwcVersion, baseName ->
       numberOfNodes = 2
       setting 'path.repo', "${buildDir}/cluster/shared/repo/${baseName}"
       setting 'xpack.security.enabled', 'false'
+      if (v.equals('8.10.0') || v.equals('8.10.1') || v.equals('8.10.2')) {
```

Comment:
I think this has to include `8.10.3` unfortunately.

Reply:
Not yet, but it will when 8.10.3 is released.

Reply:
I've added that to the checklist for the 8.10.3 release and will ping @masseyke about it too.